README.md

```diff
@@ -4,32 +4,64 @@
 
 Docker Compose file for running the entire QuAntiL environment.
 
-The fastest way to get started is using [Docker Compose](https://docs.docker.com/compose/):
+The fastest way to get started is using [Docker Compose](https://docs.docker.com/compose/).  
+[Profiles](https://docs.docker.com/compose/profiles/) are used to run different feature sets.  
 
+Defined feature sets are:  
```

Comment:
Please add some more details here: what is a feature set? what do i need profiles for? can i start it without any feature set/profile? what happens then?

Reply:
I hope it is better now